composer.json

```diff
@@ -69,7 +69,7 @@
     },
     "scripts": {
         "test:types": "phpstan analyse --ansi --memory-limit 256M",
-        "test:unit": "phpunit --colors=always -d memory_limit=1408M",
+        "test:unit": "phpunit --colors=always -d memory_limit=2048M",
```

Comment:
Should we be liberal with the memory thus loose control how much is really needed?